build/build.targets

```diff
@@ -108,8 +108,23 @@
   </Target>
 
   <Target Name="ProtoGen">
+    <!-- NB: do NOT use detectMissing for these (yet) -->
+    <!-- NB: MUST EDIT THE src\RiakClient\Messages\riak_dt.cs file -->
     <Exec WorkingDirectory="$(ProtoDir)"
-          Command="$(ProtoGenExe) -ns:RiakClient.Messages -i:%(ProtoFile.Identity) -o:$(ProtoMsgDir)\@(ProtoFile->'%(filename)').cs" />
+          Command="$(ProtoGenExe) -ns:RiakClient.Messages -i:riak_dt.proto -o:$(ProtoMsgDir)\riak_dt.cs" />
+    <Warning Text="NOTICE: *MUST* manually edit src\RiakClient\Messages\riak_dt.cs file. REMOVE the '[global::System.ComponentModel.DefaultValue(MapUpdate.FlagOp.ENABLE)]' attribute from the flag_op property of the MapUpdate class" />
```

Comment:
I thought about it, but it would take more time than I wanted to invest. Just having the warning is a very good improvement. If we ever move to using `detectMissing` for all PB messages this won't be necessary any more.